setup.py

```diff
@@ -13,7 +13,7 @@
 
 extras_fw = [
     'pyusb',
-    'serial'
+    'pyserial'
```

Comment:
i think this is the more standard one, also works on windows. imported as `serial` so a drop in replacement.